README.md

```diff
@@ -202,41 +202,262 @@ There are situations where you want to store some compiler settings in a config
     });
     ```
 
-### bsconfig.json options
+### `bsconfig.json` options
 
 These are the options available in the `bsconfig.json` file.
 
- - **project**: `string` - A path to a project file. This is really only passed in from the command line or the NodeJS API, and should not be present in `bsconfig.json` files. Prefix with a question mark (?) to prevent throwing an exception when the file does not exist.
+#### `project`
 
- - **extends**: `string` - Relative or absolute path to another `bsconfig.json` file that this `bsconfig.json` file should import and then override. Prefix with a question mark (?) to prevent throwing an exception when the file does not exist.
+Type: `string`
 
- - **cwd**: `string` - Override the current working directory
+A path to a project file. This is really only passed in from the command line or the NodeJS API, and should not be present in `bsconfig.json` files. Prefix with a question mark (?) to prevent throwing an exception when the file does not exist.
 
- - **rootDir**: `string` - The root directory of your roku project. Defaults to current directory
+#### `extends`
 
- - **stagingDir**: `string` - the folder where the transpiled files are placed. Defaults to `./out/.roku-deploy-staging`
+Type: `string`
 
- - **retainStagingDir**: `boolean` - Prevent the staging folder from being deleted after creating the package. Defaults to `false` meaning the dir is deleted every time.
+Relative or absolute path to another `bsconfig.json` file that this `bsconfig.json` file should import and then override. Prefix with a question mark (?) to prevent throwing an exception when the file does not exist. Defaults to `undefined`.
 
- - **files**: ` (string | string[] | { src: string | string[]; dest?: string })[]` - The list file globs used to find all files for the project. If using the {src;dest;} format, you can specify a different destination directory for the matched files in src.
+#### `cwd`
 
- - **outFile**: `string` -  The path (including filename) where the output file should be placed (defaults to `"./out/[WORKSPACE_FOLDER_NAME].zip"`).
+Type: `string`
 
- - **createPackage**: `boolean` - Creates a zip package. Defaults to true. This setting is ignored when `deploy` is enabled.
+If present, overrides the current working directory when invoking `bsc`. Defaults to `undefined`.
 
- - **watch**: `boolean` -  If true, the server will keep running and will watch and recompile on every file change.
+#### `rootDir`
 
- - **deploy**: `boolean` -  If true, after a successful build, the project will be deployed to the Roku specified in host.
+Type: `string`
 
- - **host**: `string` -  The host of the Roku that this project will deploy to.
+The root directory of your roku project. Defaults to the directory which contains the `bsconfig.json` file.
```

Comment:
I believe the default of `rootDir` is where `bsconfig.json` lives, not the cwd when `bsc` is invoked.

Reply:
So, it _should_ work this way, but currently it defaults to the `cwd`. I just tested and verified that I'm correct. I'd say leave it the way it's currently documented for now, and we can adjust the docs once it gets fixed in bsc.

Reply:
Thanks, updated in the new version.